climada/engine/impact.py

```diff
@@ -1389,6 +1389,114 @@ def _selected_exposures_idx(self, coord_exp):
             LOGGER.warning("No exposure coordinates match the selection.")
         return sel_exp
 
+    @classmethod
+    def concat(cls, imp_list, reset_event_ids = False):
+        """
+        Concatenate impact objects with the same exposure.
```

Comment:
Single line as docstring "title"
```suggestion
        """Concatenate impact objects with the same exposure

```